tests/Transport/Base.php

```diff
@@ -858,4 +858,12 @@ public function testBodyDataFormat() {
 		$this->assertEquals(httpbin('/post'), $result['url']);
 		$this->assertEquals(array('test' => 'true', 'test2' => 'test'), $result['form']);
 	}
+
+	public function test303GETmethod() {
+		$data = array('test' => 'true', 'test2' => 'test');
```

Comment:
Looks like this tests needs a skip check like this to prevent the build failing on PHP 5.2.

https://github.com/rmccue/Requests/blob/347016983d0938e8bc4f76924dde3163cbbfa301/tests/Transport/Base.php#L518-L521